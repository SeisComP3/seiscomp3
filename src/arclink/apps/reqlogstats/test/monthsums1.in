# DAY           TOTAL      BGR     ETHZ      GFZ     INGV     IPGP   
2016-01-01       77.0     19.0        0     28.0     15.0     15.0   
2016-01-02       83.0     19.0        0     30.0     17.0     17.0   
2016-01-03       71.0        0        0     35.0     18.0     18.0   
2016-01-04       91.0     22.0        0     32.0     18.0     19.0   
2016-01-24       81.0     19.0        0     33.0     14.0     15.0   
2016-01-25       86.0     16.0        0     39.0     16.0     15.0   
2016-01-26       21.0     21.0        0        0        0        0   
2016-01-29       75.0     17.0        0     32.0     14.0     12.0  
2016-01-30       72.0     15.0        0     30.0     13.0     14.0   
2016-01-31       72.0     19.0        0     27.0     11.0     15.0 
  
2016-02-01       94.0     19.0        0     40.0     17.0     8.0   
2016-02-02      102.0     21.0        0     41.0     21.0     9.0   
2016-02-03       94.0     17.0        0     45.0     17.0     5.0   
2016-02-04      111.0     23.0        0     51.0     18.0     9.0   
2016-02-12      117.0     21.0        0     52.0     24.0     0.0   
2016-02-13       82.0     16.0        0     38.0     14.0     4.0   
2016-02-14       74.0     15.0        0     34.0     12.0     3.0   
2016-02-15       42.0     22.0        0        0     20.0       0   
2016-11-29      144.0     22.0        0     60.0     39.0     3   
2016-11-30      108.0     23.0        0     43.0     23.0     9   
2016-12-01      109.0     22.0        0     53.0     24.0     10.0   
2016-12-02      111.0     20.0        0     55.0     22.0     14.0   
2016-12-03       88.0     18.0        0     38.0     19.0     13.0   
2016-12-04       91.0     20.0        0     38.0     21.0     12.0   
