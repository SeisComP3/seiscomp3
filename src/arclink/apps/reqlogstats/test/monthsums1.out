# DATE          TOTAL      BGR     ETHZ      GFZ     INGV     IPGP   
2016-01    729.0    167.0      0.0    286.0    136.0    140.0
2016-02    716.0    154.0      0.0    301.0    143.0     38.0
2016-11    252.0     45.0      0.0    103.0     62.0     12.0
2016-12    399.0     80.0      0.0    184.0     86.0     49.0
